### ISSUE TITLE
Getting this crash on the newest versions

### ISSUE THREAD

**Issue by breadguyyy** [MinecraftModDevelopmentMods/MMDLib#34]
https://paste.dimdev.org/mahekulati.mccrash

probably has something to do with something i disabled in the config but i cant figure out what. it crashed when i deleted the config and generated a new one too, so idk what i did

**Comment by dshadowwolf** [comment]
Yeah... I thought @YaibaToKen had handled this when it was reported during -rc1 -- basically... you've got, say, "Aluminum Brass" enabled but are missing "Copper" or something similar (like "Steel" is enabled but "Coal" is disabled)...

**Comment by breadguyyy** [comment]
Okay I think I figured it out, I'm not home so I can't test it, but I think it's because the crack hammer config started working on the newest update. I usually turn off the coal/charcoal fluid bc I like to use the Tinkers complement High Tower for steel production. I also usually turn off the crackhammer for the same reason, but in the last patch setting the config option to false on the crackhammer didn't do anything. So i guess now that it does work, steel doesn't technically have a recipe within the mod anymore.

**Comment by dshadowwolf** [comment]
...giving that a test now, as I've been unable to reproduce to see if an attempted fix worked...

**Comment by dshadowwolf** [comment]
and... coal&charcoal off, crackhammer off - no change, unable to reproduce... lets add TComp to the test environment...

**Comment by dshadowwolf** [comment]
...yeah - with all the checks that exist already in the code, I do not understand how you triggered this. I am unable to reproduce in any form in a test environment.

**Comment by breadguyyy** [comment]
well I've gotten pretty good at screwing things up in modded mc, I'll send a copy of my current config when I get home, and I'll reset them and try to reproduce it from scratch. thanks for taking the time to deal with my dumb problems lol

**Comment by breadguyyy** [comment]
okay ive reset my configs and reconfigured them and got the crash again. cant figure out whats causing it ive, double checked to make sure none of the things ive turned off are required for anything else. heres my base metals and mmdlib config files. im not using modern metals

https://paste.dimdev.org/fohajocuvu.coffeescript

**Comment by dshadowwolf** [comment]
hrm... 
```
    # Enable the molten fluid of pewter [default: true]
    B:"Enabled Pewter"=true
```
and...
```

    # Enable Pewter Items and Materials [default: true]
    B:EnablePewter=false
```
Could be it - but I've tested that setup and not triggered a crash... hrm...  I can't say - I've tried a number of things to reproduce this and have not been able to. I'm going to try a few more things and see.

**Comment by breadguyyy** [comment]
I have a lot of metals that are turned off, despite their liquid still turned on and that hasn't caused issues before. I was under the impression that disabling the metal disabled ALL of it including the liquid

**Comment by dshadowwolf** [comment]
It should - I have no idea what is happening here, though - things should just, you know, work... I know we missed some error checks in -beta7, -beta8 and -rc1 but I thought we'd gotten all those caught.

**Comment by breadguyyy** [comment]
okay finally got the chance to sit down and test all this stuff. its being caused by this config being turned to false:


    # Enable Prismarine Additions like Walls, Slabs and Pressure-plates [default: true]
    B:EnablePrismarine=false

I assume bc its used in aquamarine

edit: also molten prismarine isnt properly localized in alloying recipes (at least in my instance) its labelled as fluid.mmdlib.prismarine.name, although its fine in JEI and in inventory

**Comment by YaibaToKen** [comment]
so we're missing a check to see if a fluid is available before attempting to register an alloy using said fluid?

**Comment by breadguyyy** [comment]
i think so, i tested again and it also happens when i turn the prismarine fluid to false. i didnt think that the "prismarine additions" config turned off the fluid as well, which is why i was so confused when i got the first crash

**Comment by dshadowwolf** [comment]
dammit, I thought I'd fixed this - actually ran into an issue with Prismarine et. al. while doing testing leading up to -rc2 and thought I'd fixed it.
